frontside/lib/routers/routers.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:beamer/beamer.dart';
+import 'package:frontside/views/about/about.dart';
+import 'package:frontside/views/home/home.dart';
+
+
+
+class RouteManager{
+  static const String homeRoute = "/";
+  static const String aboutRoute = "/about";
+
+  static final routerDelegate = BeamerDelegate(
+    locationBuilder: SimpleLocationBuilder(
+        routes: {
+
+          homeRoute: (context, state) => Home(),
+          aboutRoute: (context, state) => About(),
```

Comment:
Can we rename the keys to `home` and `about`? There is no need to keep the `Route` postfix since it's already under `routes` property.

```suggestion
          home: (context, state) => Home(),
          about: (context, state) => About(),
```